Jswope00/reorg documentors docs

jswope00 opened the pull request:
In anticipation of the hackathon, I went through documentors docs and organized it similar to Educators. 
- Organize homepage by intended _action_ from the user. 
- Tag according to diataxis and role. 
- Add See also links
- Add Maintenance charts

https://docsopenedxorg--865.org.readthedocs.build/en/865/documentors/index.html

@sarina and @jesgreco let me know your feedback.

openedx-webhooks commented:
Thanks for the pull request, @jswope00!


This repository is currently maintained by `@openedx/docs-openedx-org-maintainers`.


Once you've gone through the following steps feel free to tag them in a comment and let them know that your changes are ready for engineering review.

<details><summary>:radio_button: Get product approval</summary>

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.
</details>

<details><summary>:radio_button: Provide context</summary>

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads
</details>





<details><summary>:radio_button: Get a green build</summary>

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.
</details><details>

<summary>:radio_button: Update the status of your PR</summary>

Your PR is currently marked as a draft.  After completing the steps above, update its status by clicking "Ready for Review", or removing "WIP" from the title, as appropriate.
</details>
<!-- comment:end_of_wip -->


---

<details><summary>Where can I find more information?</summary>

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)
</details>

<details><summary>When can I expect my changes to be merged?</summary>

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*
</details>
<!-- comment:external_pr -->
<!-- data: eyJkcmFmdCI6IHRydWV9 -->

sarina commented:
I think this looks great. I think I would swap the "Create an Issue" box with the "Edit an existing document" box. And I would retitle "Create an Issue" to "Report a problem with the docs"

sarina commented:
Maybe even update the title of https://docsopenedxorg--865.org.readthedocs.build/en/865/documentors/how-tos/create_github_issue.html to "Report a Docs Issue on GitHub"

jswope00 commented:
> I think this looks great. I think I would swap the "Create an Issue" box with the "Edit an existing document" box. And I would retitle "Create an Issue" to "Report a problem with the docs"

I've set the order to Create Issue, Edit an Existing Object, and Add a New Doc, which is basically in order from easiest to harder. 

I changed the title to "Report a Problem with the Documentation" to keep the word github out of that one. 

Added "free" for free account

sarina commented:
This looks good to me. @jesgreco or @Apgomeznext - do you want to weigh in?

Apgomeznext reviewed (approved):
It looks great to me!
